one-to-one-scheduler/one-to-one-scheduler.gs

```diff
@@ -626,6 +633,133 @@ function sendInvitesToMentors() {
   showInfo(TITLE_SUCCESS, summary);
 }
 
+function getRecordingLinks() {
+  let calendar = getCalendar();
+  if (!calendar) {
+    showInfo(TITLE_ERROR, MSG_CALENDAR_NOT_FOUND);
+    return;
+  }
+
+  if (!confirmCalendar(calendar)) {
+    showInfo(TITLE_ERROR, MSG_CALENDAR_REJECTED);
+    return;
+  }
+
+  let activeSheet = SpreadsheetApp.getActiveSheet();
+  let oneToOneData = get1to1Data(activeSheet);
+  if (!oneToOneData) {
+    showInfo(TITLE_ERROR, MSG_COULD_NOT_FETCH_DATA);
+    return;
+  }
+
+  let noMeetingRows = [];
+  let invalidMeetings = [];
+  let noRecordingLinks = [];
+  let numSuccess = 0;
+  let numSkipped = 0;
+  let calendarId = calendar.getId();
+
+  for (let rowIndex = 0; rowIndex < oneToOneData.length; ++rowIndex) {
+    let rowNumber = rowIndex + DATA_BEGIN_ROW;
+
+    let recordingLinkCell = activeSheet.getRange(rowNumber, RECORDING_LINK_INDEX + 1);
+    let chatLinkCell = activeSheet.getRange(rowNumber, CHAT_LINK_INDEX + 1);
+
+    let opsRequestFlag = parseInt(activeSheet.getRange(rowNumber, OPS_REQUEST_INDEX + 1).getValue());
+
+    if (opsRequestFlag !== FLAG_SEND_INVITE) {
+      // The row should be skipped
+      ++numSkipped;
+      continue;
+    }
+
+    let meetIdCell = activeSheet.getRange(rowNumber, MEETING_ID_INDEX + 1);
+    let meetId = meetIdCell.getValue().trim();
+    if (meetId === "") {
+      // Meeting is not set
+      noMeetingRows.push(rowNumber);
+      continue;
+    }
+
+    let event = getEvent(calendar, meetId);
+    if (!event) {
+      invalidMeetings.push(rowNumber);
+      continue;
+    }
+
+    let eventResource = Calendar.Events.get(calendarId, meetId);
+    if (!eventResource) {
+      console.log("Event could not be retrieved - " + calendarId + ", " + meetId);
+      invalidMeetings.push(rowNumber);
+      continue;
+    }
+
+    let eventAttachments = eventResource.attachments;
+    
+    if (!eventAttachments) {
+      recordingLinkCell.setValue(NOT_FOUND);
+      chatLinkCell.setValue(NOT_FOUND);
+      noRecordingLinks.push(rowNumber);
+      continue;
+    } else {
+      for (let i = 0; i < eventAttachments.length; ++i) {
+        let linkUrl = eventAttachments[i].fileUrl;
+        let queryIndex = linkUrl.indexOf("?");
+        linkUrl = linkUrl.substring(0,queryIndex);
+        if (eventAttachments[i].mimeType === "video/mp4") {
```

Comment:
`const RECORDING_MIME_TYPE = "video/mp4";`
`const CHAT_MIME_TYPE = "text/plain";`
`const QUERY_DELIMITER = "?";`

were added and used instead of hardcoded strings